### DISCUSSION TITLE
FYI for everyone if you enable the YouTube premium logo option and restart the app it’ll start crashing and won’t open.

### DISCUSSION THREAD

**Discussion by Shaukariya** [arichornlover/uYouEnhanced#209] [General]
Title.

**Comment by neokatin** [comment]
That's nice how do you fix it tho if you didn't know about that😭?

**Reply by Shaukariya** [reply]
> That's nice how do you fix it tho if you didn't know about that😭?

Ihnc I just downgraded to the prev version, gonna wait for the next release.

**Comment by arichornlover** [comment]
![IMG_2486](https://github.com/arichorn/uYouPlusExtra/assets/78001398/57b9c256-b6de-4170-801d-c72e477ff1f8)

I changed it again in the latest release. Check out these results.
